git_repo/repo.py

```diff
@@ -210,16 +205,15 @@ def set_repo_slug(self, repo_slug, auto=False):
         self.repo_slug = EXTRACT_URL_RE.sub('', repo_slug) if repo_slug else repo_slug
         self._auto_slug = auto
         if not self.repo_slug:
-            self.user_name = None
+            self.namespace = None
             self.repo_name = None
         elif '/' in self.repo_slug:
             # in case a full URL is given as parameter, just extract the slug part.
-            self.user_name, self.repo_name, *overflow = self.repo_slug.split('/')
-            if len(overflow) != 0:
-                raise ArgumentError('Too many slashes.'
-                                    'Format of the parameter is <user>/<repo> or <repo>.')
+            #self.namespace, self.repo_name, *overflow = self.repo_slug.split('/')
```

Comment:
as a todo note: don't forget to remove that commented line when you'll be done 😉